src/components/Images.vue

```diff
@@ -15,9 +15,18 @@
         :paging="true"
       >
         <template #header-middle>
+          <div v-if="supportsNamespaces">
+            <label>Image Namespace:</label>
+            <select class="select-namespace" :value="selectedNamespace" @change="handleChangeNamespace($event)">
+              <option v-for="item in imageNamespaces" :key="item" :value="item" :selected="item === selectedNamespace">
+                {{ item }}
+              </option>
+            </select>
+          </div>
```

Comment:
The rendered layout is off. Can we align the elements in the visual layout?

<img width="1052" alt="Untitled" src="https://user-images.githubusercontent.com/62991/135668965-1a7d46a8-8f8e-4e0e-9911-079749262735.png">

Reply:
Addressing this multi-complaint comment:
1. Added a new state, `VM_STARTED`, to indicate that the VM is ready but K8s isn't.
2. Dropped names referring to `kim`
3. Alignment fixed in a separate commit. I added a comment noting that I don't think we can use a CSS grid to do this, because the Filter field goes in one Vue.js component `slot`s, and the other widgets go in other slots.